k8s-openapi-codegen/src/main.rs

```diff
@@ -41,6 +41,35 @@ impl std::fmt::Debug for Error {
 	}
 }
 
+struct RequestedVersion {
+	ver: supported_version::SupportedVersion,
+	overriden_spec_url: Option<String>
+}
+
+fn parse_requested_versions(spec: &str) -> Vec<RequestedVersion> {
+	let mut res = Vec::new();
+	for item in spec.split(',') {
+		let mut it = item.split('=');
+		let name = it.next().expect("empty requested version");
+		let url = it.next();
+		let mut found_version = false;
+		for ver in supported_version::ALL.iter().copied() {
+			if ver.name() == name {
+				res.push(RequestedVersion {
+					ver,
+					overriden_spec_url: url.map(ToString::to_string)
+				});
+				found_version = true;
+				break;
+			}
+		} 
+		if !found_version {
+			panic!("unknown requested version");
```

Comment:
Return an `Err()` like the rest of the code, and include `{:?}` of `found_version` in the message.